data/planner.js

```diff
@@ -0,0 +1,21 @@
+module.exports = [
+
+    {
+        task:'Create data tables',
+        complete: true,
+        user_id: 1
+    },
+    {
+        task:'Back-end list server',
+        complete: false,
+        user_id: 1
+    },
+    {
+        task:'Front-end list app',
+        complete: false,
+        user_id: 1
+    },
+    {
+        task:'D'
+    }
```

Comment:
This object may be messing up your setup-db because it doesn't have a complete and user_id